client/extensions/woocommerce/state/ui/products/variations/selectors.js

```diff
@@ -94,5 +94,5 @@ export function getProductVariationsWithLocalEdits( state, productId, siteId = g
 		return { ...variation, ...update };
 	} ) );
 
-	return ( creates || variations ? [ ...creates || [], ...updatedVariations || [] ] : undefined );
+	return creates || variations ? [ ...(creates || []), ...(updatedVariations || []) ] : undefined;
```

Comment:
I'm pretty sure this came from the code printer. I didn't adjust anything related to parentheses, for example. it was something I was willing to live with in order to clean up the rest, knowing that `prettier` will eventually take care of it.